### ISSUE TITLE
CssSyntaxError, Missed semicolon on Gutenberg 17.0.2

### ISSUE THREAD

**Issue by csabarakasz** [Automattic/wp-calypso#84364]
### Quick summary

The user is not able to create a new post, because the Editor won't load but shows an error, and if you copy the error message it will show something similar:

```
CssSyntaxError: <css input>:15:107: Missed semicolon
    at g.error ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:94112)
    at e.exports.checkMissedSemicolon ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:115103)
    at e.exports.decl ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:117150)
    at e.exports.other ([domain.com]wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:119091)
    at e.exports.parse ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:119430)
    at l ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:113525)
    at new y ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:97078)
    at s.process ([domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:1:124965)
    at [domain.com]/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=7a6a1df140c4fe7830e1:18:19596
    at Array.map (<anonymous>)
```
    
Deactivating Gutenberg solves the issue. 

### Steps to reproduce

I couldn't replicate this on a test site yet, but on user's site:

- Make sure Gutenberg is active
- Click to create a new post
- An error message will show that it failed and offers to Try again or Copy error message.

### What you expected to happen

New post would open with Gutenberg

### What actually happened

Error message

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

Workaround at the moment: deactivate Gutenberg 17.0.2

### Platform (Simple and/or Atomic)

Atomic

**Comment by csabarakasz** [comment]
7339564-zd-a8c

**Comment by csabarakasz** [comment]
From _WP 6.4 + Gutenberg are causing critical errors when editing posts on Atomic sites_ https://github.com/Automattic/wp-calypso/issues/83986, these reports appear to belong here:

7268530-zd-a8c
7322181-zd-a8c
7318070-zd-a8c
7318641-zd-a8c
7307909-zd-a8c
7322181-zd-a8c

**Comment by villanovachile** [comment]
7342285-zd-a8c running theme `bromley` on a Simple Site with the following Editor error:

```
CssSyntaxError: <css input>:17:24: Missed semicolon
    at g.error (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:94112)
    at e.exports.checkMissedSemicolon (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:115103)
    at e.exports.decl (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:117150)
    at e.exports.other (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:118701)
    at e.exports.parse (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:119430)
    at l (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:113525)
    at new y (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:97078)
    at s.process (https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:1:124965)
    at https://s0.wp.com/wp-content/plugins/gutenberg-core/v17.0.2/build/block-editor/index.min.js?m=1699889169i&ver=7a6a1df140c4fe7830e1:18:19596
    at Array.map (<anonymous>)
```

@fullofcaffeine  just wanted to loop you in that this release is also affecting at least one theme (even if it is retired) on dotcom, and could potentially be affecting others.

**Comment by zaguiini** [comment]
Thanks for reporting. https://github.com/WordPress/gutenberg/pull/56093 will fix it.

**Comment by zaguiini** [comment]
https://github.com/WordPress/gutenberg/pull/56093 is merged. Once the PR is deployed, @csabarakasz can you verify whether the problem still happens?

**Comment by dolgelukkig** [comment]
7346235-zd

**Comment by liviopv** [comment]
Pinged here p1700732871460589-slack-CBTN58FTJ to possibly include this in a 17.1.1 release instead of waiting for 17.2

**Comment by liviopv** [comment]
📌 REPRODUCTION RESULTS
Already fixed by Gutenberg 17.1.4, please reopen if you run into it again.

📌 ACTIONS
- Triaged
